#!bin/usr/python3
def raise_exception():
    return "a" / "b"
